src/components/index.js

```diff
@@ -324,6 +324,12 @@ export default class Rnd extends Component {
   }
 
   render() {
+    const innerStyle = {
+      ...boxStyle,
+      zIndex: this.state.z,
+      ...this.props.disableDragging && { cursor: 'normal' },
```

Comment:
Please fix to below.

`this.props.disableDragging ? { cursor: 'normal' } : {}`